certification/internal/shell/mounted_engine.go

```diff
@@ -27,16 +27,17 @@ func (e *MountedCheckEngine) ExecuteChecks() error {
 	// We want to know the time just for the check itself, so reset checkStartTime
 	checkStartTime := time.Now()
 
-	// run the validation
-	pass, err := containerutil.RunInsideContainerFS(podmanEngine, targetImage, e.Check.Validate)
+	// If the target is a bundle, we need to run the image filesystem mount logic instead
```

Comment:
This comment doesn't seem to make sense anymore.

Reply:
@bcrochet fixed